cpp/src/gandiva/precompiled/string_ops.cc

```diff
@@ -1246,6 +1258,52 @@ const char* convert_fromUTF8_binary(gdv_int64 context, const char* bin_in, gdv_i
   return ret;
 }
 
+FORCE_INLINE
+const char* convert_replace_invalid_fromUTF8_binary(
+    gdv_int64 context, const char* text_in, gdv_int32 text_len,
```

Comment:
The aliases gdv_int64, gdv_int32, etc are only used to make it easy to write macros. In normal function definitions it is better to use normal types int64_t, int32_t etc. for better readability. (Can replace instances of gdv_* types in other functions with the actual types in a separate PR?)

Reply:
Sure, I will correct it for this function and then create a "story" to correct it on JIRA